### DISCUSSION TITLE
How do I view my repository on a browser to see what it looks like

### DISCUSSION THREAD

**Discussion by Julie28K** [community/community#29944] [New to GitHub]
Hello!  I"m having the hardest time trying to view my repository.  I just want to copy its url and see the end result in a browser.
[
](https://github.com/Julie28K/Tea-Cozy-Project.git)

**Comment by airtower-luna** [comment]
> I just want to copy its url and see the end result in a browser.

What do you mean by "end result"? Browser sounds like the data in your repository might be for a website.

**Reply by Julie28K** [reply]
I want to preview the work and see how it looks, not just see the code.

**Reply by Julie28K** [reply]
Thank you for your help!

**Reply by airtower-luna** [reply]
> Its suppose to look like a website.

That's kind of key. If it's a static website (or a static website built using some tool) you can [deploy it to Pages](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages) to see it as a website. If it's a dynamic application (e.g. with Django, PHP, …) the answer is no, not possible on GitHub.

**Comment by cheloniixd** [comment]
Does your repo contain an index.html? If so you can just run from a live server (most likely one built into the code editor) or start webpack and open localhost300... I don't know exactly what you mean by (look at ur work) so I'm assuming its a small js project or something.

**Reply by Julie28K** [reply]
So its a project I built in VS Code - A Codecademy project, I uploaded my work into Github.  Its suppose to look like a website.  I'll go back into my VS and see if I can preview it that way.

**Reply by Julie28K** [reply]
I appreciate your help!

**Reply by Julie28K** [reply]
I found it thank you!

**Comment by SoloMacherYT** [comment]
I was just looking for what you are.
I don't know how professionals do it but I got a way through that worked for me.

TRY THIS:
Step 1: Open your GitHub repository that contains a html file that you want to see the output in the browser
Step 2: Click and Open the html file that you want eg index.html
Step 3: Then click on the ... more action button above the history button and click on copy permanent link. The GitHub link will be copied
Step 4: Then go to Raw.githack.com and paste the link into the input box
Step 5: Copy the URL in the production field and paste it into the browser
